tests/strategies/test_strategies.py

```diff
@@ -407,13 +421,13 @@ def _custom_check(series: pd.Series) -> pd.Series:
 )
 def test_series_strategy(data):
     """Test SeriesSchema strategy."""
-    series_schema = pa.SeriesSchema(pa.Int, pa.Check.gt(0))
+    series_schema = pa.SeriesSchema(pa.Int(), pa.Check.gt(0))
```

Comment:
is `pa.Int()` and more generally `pa.[DataType]()` the way the public-facing pandera data types are used? i.e. could I do `pa.Int` still?

Reply:
It depends on the situation. Externally, i.e. in schemas __init__ you can pass the class. It will be handed to pandas_engine.Engine.dtype which will instantiate it, if there is a default constructor (won't work for parametrized dtypes with no defaults). Internally, we need an instance to call coerce and check methods. 

TLDR User facing code doesn't change. I will have a pass and standardize to DataType class for schema creation.

Reply:
> TLDR User facing code doesn't change

💯